src/monster.cpp

```diff
@@ -932,7 +926,7 @@ void Monster::onThinkDefense(uint32_t interval)
 		}
 	}
 
-	if (!isSummon() && summons.size() < mType->maxSummons) {
+	if (!isSummon() && summons.size() < mType->maxSummons && hasFollowPath) {
```

Comment:
I have not tested them throughly to be honest. But i have tested and everything run smooth.

Any tests you want me to do, since i tried the changes and they behaved like they were coded todo.